jsk_baxter_robot/baxtereus/baxter-interface.l

```diff
@@ -252,8 +252,8 @@
      (progn
        (unless tm (setq tm 3000))
        (send-super* :angle-vector-motion-plan av :ctype ctype :move-arm move-arm :total-time tm
-                    :start-offset-time start-offset-time :clear-velocities clear-velocities
-                    :use-torso nil args))
+                    :start-offset-time (if start-offset-time start-offset-time start-time)
```

Comment:
`start-time` is `0`, but is it OK?

Reply:
The robot is correctly moved by `(send *ri* :angle-vector (send *baxter* :angle-vector) 1000)` in this configuration.
Default value of `start-offset-time` of `:angle-vector-motion-plan` is also `0`:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus_moveit/euslisp/robot-moveit.l#L527
 
The only method having non-zero `start-offset-time` as default is `:angle-vector-make-trajectory`:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus_moveit/euslisp/robot-moveit.l#L455
You added this to follow `:angle-vector-sequence`, right?
https://github.com/jsk-ros-pkg/jsk_pr2eus/commit/0135eacd18d972852d58fca6c476f4fa5db4005a
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus/robot-interface.l#L406

So, what is problem seems that `:angle-vector-sequence` has non-zero `start-offset-time`, which is different from `:angle-vector`...